### PR TITLE
Bugfix. Crash when printing OpenSSL decryption errors with no e2ee mnemonic,

### PR DESCRIPTION
**Pull request by allexzander**
Signed-off-by: allexzander <blackslayer4@gmail.com>

### CODE REVIEW

File: src/libsync/clientsideencryption.cpp [hunk 1]
```diff
@@ -706,8 +706,8 @@ QByteArray decryptStringAsymmetric(EVP_PKEY *privateKey, const QByteArray& data)
     QByteArray out(outlen, '\0');
 
     if (EVP_PKEY_decrypt(ctx, unsignedData(out), &outlen, (unsigned char *)data.constData(), data.size()) <= 0) {
-        qCInfo(lcCseDecryption()) << "Could not decrypt the data.";
-        ERR_print_errors_fp(stdout); // This line is not printing anything.
+        qCCritical(lcCseDecryption()) << "Could not decrypt the data.";
+        qCCritical(lcCseDecryption()) << handleErrors();
```
Comment (mgallien) [inline src/libsync/clientsideencryption.cpp:710]: Please call `handleErrors` before the call to QCCritical such that it is always done even if qCCritical is disabled

Comment (allexzander) [reply src/libsync/clientsideencryption.cpp]: @mgallien fixed

**Review by mgallien — APPROVED** [review]

**Review by FlexW — APPROVED** [review]

**Pull request merged**